### PR TITLE
[wmco] Remove constant for metrics resources name 

### PR DESCRIPTION
**Pull request by VaishnaviHire**
This commit removes the constant used as name for  metrics resources.
The name for metrics resources is set by 'metrics' package of operator-sdk
and is equivalent to the name of the metrics service. The changes include
retrieving the name set by the 'metrics' package and using it for accessing
the resources.

### CODE REVIEW

**Review by sebsoto — APPROVED** [review]

File: pkg/controller/windowsmachine/metrics/metrics.go [hunk 1]
```diff
@@ -84,6 +84,9 @@ func Add(ctx context.Context, cfg *rest.Config, namespace string) error {
 		return errors.Wrap(err, "could not create metrics Service")
 	}
 
+	// the name for the metrics resources is set during creation of metrics service and is equivalent to the service name
+	windowsMetricsResource = service.Name
```
Comment (sebsoto) [inline pkg/controller/windowsmachine/metrics/metrics.go:88]: Does service have a `GetName()` method that "should" be used?

**Review by aravindhp — APPROVED** [review]
Thanks for fixing this, @VaishnaviHire 
/lgtm

**Pull request merged**